src/atoms/LinkBarSearchField.jsx

```diff
@@ -1,45 +1,62 @@
-import React, { Fragment } from 'react';
+/* eslint-disable jsx-a11y/label-has-for, jsx-a11y/label-has-associated-control */
+import React from 'react';
 import PropTypes from 'prop-types';
 import styled from '@emotion/styled';
 import { css } from '@emotion/core';
 
-import { getColor, getVariable } from '../utils';
+import { getColor, getVariable, linkBarElementSizes } from '../utils';
 import { LinkBarElementBase } from './LinkBarElementBase';
-import { FontIcon } from './FontIcon';
+import { SvgIcon } from './SvgIcon';
+
+const AbsoluteIcon = styled(SvgIcon)`
+	${linkBarElementSizes}
```

Comment:
The icon will have the same sizing as LinkBar* elements - specifically the LinkBarSearchField

Reply:
as a result of this ${linkBarElementSizes} we get css block {margin, padding, font-size, line-height}, right? I ask to check myself

Reply:
yes, that is correct.

more specifically, linkBarElementSizes is a delta function that will be executed when the interpolation is resolved. This delta function returns the margin, padding, font-size and line-height css.